src/formio/components/DateField.stories.js

```diff
@@ -38,5 +41,61 @@ export const DateField = {
   args: {
     key: 'date',
     label: 'Datum',
+    extraComponentProperties: {
+      format: 'dd-MM-yyyy',
+      placeholder: 'dd-mm-yyyy',
+      enableTime: false,
+      datePicker: {
+        minDate: null,
+        maxDate: null,
+      },
+    },
+  },
+  play: async ({canvasElement}) => {
+    const canvas = within(canvasElement);
+
+    const dateInput = canvas.getByRole('textbox');
+
+    userEvent.type(dateInput, '06-06-2006');
```

Comment:
```suggestion
    await userEvent.type(dateInput, '06-06-2006');
```
this is async iirc

Reply:
From the docs I saw that it is async if you use the delay option (which then adds a delay between each key pressed).

However, with the async behaviour, it looks like the flatpickr instance is reinitialised and so you loose what was filled in :S

Reply:
towel, ring... etc :) let's leave it at that